src/EditorFeatures/CSharpTest/SignatureHelp/AttributeSignatureHelpProviderTests.cs

```diff
@@ -356,6 +356,35 @@ class D
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        [WorkItem(12544, "https://github.com/dotnet/roslyn/issues/12544")]
+        public async Task TestAttributeWithOverriddenAndNewProperties()
+        {
+            var markup = @"
+class BaseAttribute : System.Attribute
+{
+    public virtual int foo { get; set; }
+    public int bar { get; set; }
+}
+class DerivedAttribute : BaseAttribute
+{
+    public override int foo { get; set; }
+    public new int bar { get; set; }
+
+}
+[[|Derived($$|])]
+class D
+{
+}";
+
+            var expectedOrderedItems = new List<SignatureHelpTestItem>();
+            expectedOrderedItems.Add(new SignatureHelpTestItem($"DerivedAttribute({CSharpFeaturesResources.Properties}: [bar = int], [foo = int])", string.Empty, string.Empty, currentParameterIndex: 0));
+
+            // TODO: Bug 12319: Enable tests for script when this is fixed.
```

Comment:
This comment doesn't seem to make sense - it nothing else mentions script, and the number is actually a PR, not a bug.